### PR TITLE
Markus' retirement

### PR DESCRIPTION
**Pull request by mrks**

### CODE REVIEW

**Comment by Bouncner** [comment]
:cry:

**Comment by mrks** [comment]
CI doesn't approve either. Guess that's settled then. Another six years puts me at 36, just shy of retirement.

**Comment by Bensk1** [comment]
> CI doesn't approve either. Guess that's settled then. Another six years puts me at 36, just shy of retirement.

It even gives a reason: _Timeout exceeded._

**Review by mrzzzrm — APPROVED** [review]
Should've revoked my approval rights, people.

File: README.md [hunk 1]
```diff
@@ -145,17 +145,16 @@ When trying to optimize the time spent building the project, it is often helpful
...
 - Jan Kossmann
-- Markus Dreseler
 - Martin Boissier
 - Stefan Halfpap
 
-
 Contact: firstname.lastname@hpi.de
 
-## Contributors
+## Maintainer emeritus
```
Comment (Bensk1) [inline README.md:154]: `Assertor Emeritus`?

https://en.wiktionary.org/wiki/assertor

**Review by Bensk1 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (mrks) [reply README.md:154]: The first Google results just give me Star Wars pictures...

**Comment by ArneMayer** [comment]
Not Approved!

**Pull request merged**